include/rocksdb/sst_file_writer.h

```diff
@@ -43,8 +43,12 @@ struct ExternalSstFileInfo {
 // All keys in files generated by SstFileWriter will have sequence number = 0
 class SstFileWriter {
  public:
+  // User can pass `column_family` to specify that the the generated file will
+  // be ingested into this column_family, note that passing nullptr means that
+  // the column_family is unknown.
   SstFileWriter(const EnvOptions& env_options, const Options& options,
-                const Comparator* user_comparator);
+                const Comparator* user_comparator,
+                ColumnFamilyHandle* column_family = nullptr);
```

Comment:
I think we should allow them to simply pass in column family ID. For some systems, this file may be generated from some other places, e.g. data warehouse. They don't have access to the handle but they may know the column family ID.